File: MachineProblem/Easy/Easy3.cpp
#include <iostream>
using namespace std;
int main() {
    int n; if (!(cin >> n)) return 0;
    long long fact = 1;
    for (int i = 1; i <= n; ++i) fact *= i;
    cout << fact << '\n';
    return 0;
}

File: MachineProblem/Easy/Easy3.java
/*
    EASY 3 – Factorial Calculator
    Goal: Compute factorial of n (n <= 10).
    Input: single integer n (e.g. "5")
    Output: factorial value (e.g. "120")
*/
import java.util.*;
public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        if (!s.hasNextInt()) return;
        int n = s.nextInt();
        long fact = 1;
        for (int i = 1; i <= n; i++) fact *= i;
        System.out.println(fact);
    }
}
